psfr/psfr.py

```diff
@@ -325,10 +333,16 @@ def base_stacking(star_list, mask_list):
     """
     star_stack_base = np.zeros_like(star_list[0])
     weight_map = np.zeros_like(star_list[0])
+
     for i, star in enumerate(star_list):
         star_stack_base += star * mask_list[i]
         weight_map += mask_list[i] * np.sum(star)
+
+    ###code can't handle situations where there is never a non-zero pixel
+    weight_map[weight_map == 0] = 1e-6
```

Comment:
1e-6 might a bit too high depending on some applications. What about 1e-12? This is still in the range of floats to be handled numerically stable

Reply:
Just saw this, I'll change it.